website/content/en/v0.32/troubleshooting.md

```diff
@@ -637,6 +637,18 @@ This typically occurs when the node has not been considered fully initialized fo
 
 This error indicates that the `vpc.amazonaws.com/pod-eni` resource was never reported on the node. If you've enabled Pod ENI for Karpenter nodes via the `aws.enablePodENI` setting, you will need to make the corresponding change to the VPC CNI to enable [security groups for pods](https://docs.aws.amazon.com/eks/latest/userguide/security-groups-for-pods.html) which will cause the resource to be registered.
 
+### AWS Node Termination Handler (NTH) interactions
+An identified potential cause for a continuous cycle of nodes being provisioned and immediately deprovisioned by Karpenter, can stem from a functionality that exists within AWS Node Termination Handler (NTH) called Capacity Rebalancing.
```

Comment:
I'd reframe this in terms of what users might do to overcome limitations of Karpenter and what Karpenter recommends. The project hasn't implemented support for spot rebalance recommendations as a termination option because it has the potential to create a large amount of node churn and we think that it's a net negative for a lot of users.

```suggestion
Karpenter doesn't currently support draining and terminating on spot rebalance recommendations. Users who want support for both drain and terminate on spot interruption as well as drain and termination on spot rebalance recommendations may install Node Termination Handler (NTH) on their clusters to support this behavior.

These two components do not share information between each other, meaning if you have drain and terminate functionality enabled on NTH, NTH may remove a node for a spot rebalance recommendation. Karpenter will replace the node to fulfill the pod capacity that was being fulfilled by the old node; however, Karpenter won't be aware of the reason that that node was terminated. This means that Karpenter may launch the same instance type that was just deprovisioned, causing a spot rebalance recommendation to be sent again. This can result in very short-lived instances where NTH continually removes nodes and Karpeneter re-launches the same instance type over and over again.

Karpenter doesn't recommend reacting to spot rebalance recommendations when running Karpenter with spot nodes; however, if you absolutely require this functionality, note that the above scenario is possible. 
```

Reply:
Is there a reason to recommend running NTH at all here if we Karpenter already has native interruption support for spot interruptions? I think at this point, the only value prop for NTH over Karpenter's native handling is the rebalance recommendation feature; however, we don't recommend that users use that.

Reply:
There are instances where users have both. I know quite a few that are using both for particular use cases. so it makes sense to warn of this potential disruption.